Booting/linux-bootstrap-3.md

```diff
@@ -175,7 +184,7 @@ for (card = video_cards; card < video_cards_end; card++) {
 
 but `video_cards` not declared anywhere. Answer is simple: Every video mode presented in the x86 kernel setup code has definition like this:
 
-```C
+```c
```

Comment:
small `c` supports highlighting on other md flavours. I could change them back if you wish